playbooks/keyholders/NEW_SIGNER.md

```diff
@@ -139,9 +139,9 @@ Like testing, this will create a new directory under `ceremony/$DATE/keys/` with
 
 ### Signing
 
-When prompted in the Slack channel, begin signing root and targets metadata. Run:
+When prompted in the Slack channel, begin signing root and targets metadata. If the ceremony started on a prior date, you should add a repository reference by setting `REPO`. Otherwise, for same-day signing, you may omit to default to the current date. Run:
```

Comment:
```suggestion
When prompted in the Slack channel, begin signing root and targets metadata. If the ceremony started on a prior date (or the date in your timezone does not match the date in the orchestrator's timezone), you should add a repository reference by setting `REPO`. Otherwise, for same-day signing, you may omit to default to the current date. Run:
```